roofit/roofitcore/CMakeLists.txt

```diff
@@ -499,7 +499,7 @@ ROOT_STANDARD_LIBRARY_PACKAGE(RooFitCore
 )
 
 if (roofit_multiprocess)
-  target_link_libraries(RooFitCore PRIVATE RooFitMultiProcess)
+  target_link_libraries(RooFitCore PUBLIC RooFitMultiProcess)
```

Comment:
Please don't do this change: we don't want to expose RooFitMultiProcess to the public, so this needs to be `PRIVATE`.